src/MOI_wrapper.jl

```diff
@@ -396,12 +391,19 @@ function _constraint_contains(model::Optimizer, v::VI, ci::CI{F,S}) where {F <:
     if func isa MOI.SingleVariable
         return v == func.variable
     elseif func isa MOI.ScalarAffineFunction{Float64}
-        for term in func.terms
-            if term.variable_index == v
-                return true
-            end
-        end
-        return false
+        indices = [term.variable_index for term in func.terms]
+        return v in indices
+    end
+    return false   # default
+end
+
+function _constraint_contains(model::Optimizer, v::VI, ci::CI{F,S}) where {F <: SUPPORTED_VECTOR_FUNCTIONS, S <: SUPPORTED_VECTOR_SETS}
```

Comment:
Rewrite `_constraint_contains` as to use dispatch on the `F` type:
```julia
 function _constraint_contains(model::Optimizer, v::VI, ci::CI{F,S}) where {F <: MOI.VectorOfVariables, S <: SUPPORTED_VECTOR_SETS}
...
 function _constraint_contains(model::Optimizer, v::VI, ci::CI{F,S}) where {F <: MOI.VectorAffineFunction, S <: SUPPORTED_VECTOR_SETS}
...
```

Reply:
will it improve efficiency?

Reply:
coz `SUPPORTED_VECTOR_FUNCTIONS` is nothing more than a Union of both

Reply:
Yes, make all the difference for efficiency.
This kind of statement is terrible: `func isa MOI.VectorOfVariables` for performant julia.
Doing that as @matbesancon suggested is the way to go.

Reply:
ok fixed it

Reply:
will keep it in mind for future